src/oneelement.jl

```diff
@@ -432,6 +432,14 @@ permutedims(o::OneElementMatrix) = OneElement(o.val, reverse(o.ind), reverse(o.a
 permutedims(o::OneElementVector) = reshape(o, (1, length(o)))
 permutedims(o::OneElement, dims) = OneElement(o.val, _permute(o.ind, dims), _permute(o.axes, dims))
 
+# unique
+function unique(O::OneElement)
+    v = getindex_value(O)
+    len = iszero(v) ? 1 : min(2, length(O))
+    OneElement(getindex_value(O), len, len)
```

Comment:
Better to return a `Fill`?

Reply:
It can't be a `Fill` in general, as there may be two values: `0` and the `val`.